Flex_Reg_CLI/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+#-----------------------------------------------------------------------------
+set(MODULE_NAME Flex_Reg_CLI)
+
+SlicerMacroBuildScriptedCLI(
+  NAME ${MODULE_NAME}
+)
+
+set(FOLDER_LIBRARY Method)
```

Comment:
There is a CMake error because multiple CMake paths prdouce this same `Method` package. We must rename the package to avoid the conflict. I'll work on this.

Reply:
The conflict is:

- `Flex_Reg_CLI/Method/` -> `FlexReg_CLI/FlexReg_Method/`
- `AutoMatrix/Method/` -> `AutoMatrix/AutoMatrix_Method/`

I also notice this which doesn't currently cause a conflict but should be renamed to avoid a conflict in the future.

- `MRI2CBCT/utils/` -> `MRI2CBCT/MRI2CBCT_utils/`

Reply:
I've renamed the folder from Method to Method_FlexRegCli :)

Reply:
In 201c4b9 I also renamed the `utils` and `AutoMatrix/Method` packages.

I also renamed `Method_FlexReg` to `FlexReg_Method` to be consistent with the other modules, eg. `AREG_Method`